helpers/github_helpers.rb

```diff
@@ -5,8 +5,10 @@ def build_contributors(repo)
     return unless config[:github_auth]
     @@github = Github.new do |c|
       c.basic_auth = config[:github_auth]
+      c.connection_options = {headers: {"X-GitHub-OTP" => config[:github_tfa]}}
     end
-    @@repo_contributors = @@github.repos.contributors 'ind9', repo
+    print(", " + repo)
```

Comment:
The contributors are fetched by the server. If I don't have auth in dev time, I don't see the contributors list.